cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverTest.java

```diff
@@ -108,7 +109,7 @@ public static void startLogSaver() throws Exception {
     cConf.set(LoggingConfiguration.LOG_BASE_DIR, logBaseDir);
 
     injector = KAFKA_TESTER.getInjector();
-    impersonator = injector.getInstance(Impersonator.class);
+    impersonator = injector.getInstance(DefaultImpersonator.class);
```

Comment:
This is unit test, doesn't matter. However, I think it's better not to do this. Will change